### PR TITLE
modified LE related examples

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
modified LE related examples

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/logical_interconnect_groups.py [hunk 1]
```diff
@@ -34,193 +34,311 @@
...
+                    }
+                }
+            ],
+            "networkType": "Ethernet",
+            "ethernetNetworkType": "ImageStreamer",
+            "name": "deploy"
+        },
+        {
+            "networkUris": [
+                "/rest/ethernet-networks/26cb6480-23d0-4391-b481-ae524dd54f67"
```
Comment (chebroluharika) [inline examples/logical_interconnect_groups.py:259]: if possible try to make use of data parameter of ethernet network resource to set this URI

File: examples/logical_interconnect_groups.py [hunk 2]
```diff
@@ -34,193 +34,311 @@
...
+                "permittedInterconnectTypeUri": interconnect_type2_uri,
+                "enclosureIndex": 3
             }
         ]
-    }
+    },
+    "uplinkSets": [
+        {
+            "networkUris": [
+                "/rest/ethernet-networks/87650354-afb2-49b8-adc2-edd759ddd5d2"
```
Comment (chebroluharika) [inline examples/logical_interconnect_groups.py:172]: same as below

Comment (VenkateshRavula) [reply examples/logical_interconnect_groups.py]: done

File: examples/logical_interconnect_groups.py (hunk 1 above)
Comment (VenkateshRavula) [reply examples/logical_interconnect_groups.py]: done

**Review by chebroluharika — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**